processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/MixedCaseValidatorGenerator.java

```diff
@@ -70,18 +70,90 @@ private static TypeSpec generateValidator(GtfsFileDescriptor fileDescriptor) {
       if (!mixedCaseField.mixedCase()) {
         continue;
       }
+      String test = "blah";
```

Comment:
This is probably a left over from testing/debugging the class; please delete the line.